src/modules/fancyzones/editor/FancyZonesEditor/App.xaml.cs

```diff
@@ -21,6 +21,32 @@ namespace FancyZonesEditor
     /// </summary>
     public partial class App : Application
     {
+        // Localizable strings
+        private const string CrashReportMessageBoxTextPart1 = "Error logged to ";
+        private const string CrashReportMessageBoxTextPart2 = "Please report the bug to ";
+        private const string CrashReportMessageBoxCaption = "Error";
+
+        // Non-localizable strings
+        private const string CrashReportMessageBoxCaptionFZEditor = "FancyZones Editor ";
```

Comment:
this should just be app title in the RESX

Reply:
Moved to resource file.